src/databricks/labs/ucx/framework/tasks.py

```diff
@@ -76,18 +78,51 @@ def trigger(*argv):
     if "config" not in args:
         msg = "no --config specified"
         raise KeyError(msg)
-
     task_name = args.get("task", "not specified")
+    parent_run_id = args.get("parent_run_id", "unknown_run_id")
+    job_id = args.get("job_id")
     if task_name not in _TASKS:
         msg = f'task "{task_name}" not found. Valid tasks are: {", ".join(_TASKS.keys())}'
         raise KeyError(msg)
 
     current_task = _TASKS[task_name]
     print(current_task.doc)
 
+    config_path = Path(args["config"])
+    cfg = WorkspaceConfig.from_file(config_path)
     _install()
 
-    cfg = WorkspaceConfig.from_file(Path(args["config"]))
-    logging.getLogger("databricks").setLevel(cfg.log_level)
-
-    current_task.fn(cfg)
+    logger = logging.getLogger("databricks")
+    logger.setLevel(cfg.log_level)
+
+    filepath = pathlib.Path(config_path).parent
+    logpath = filepath.joinpath(f"logs/{current_task.workflow}/{parent_run_id}")
```

Comment:
```suggestion
    logpath = config_path.parent / "logs" / current_task.workflow / parent_run_id
```

Simpler with this API, right?